src/esmock.d.ts

```diff
@@ -1,71 +1,85 @@
-type mocks = Record<string, any>
+type MockMap = { [specifier: string]: any };
 
-type opts = {
-  strict?: boolean | undefined,
-  purge?: boolean | undefined,
-  isModuleNotFoundError?: boolean | undefined
-}
+type Options = {
```

Comment:
The consumer can alias a shorter name if desired. Clear semantics are useful.

Reply:
okay thank you